python/cudf/cudf/core/dataframe.py

```diff
@@ -3733,31 +3733,31 @@ def agg(self, aggs, axis=None):
                 dict of axis labels specified operations per column
                 function name: must pass to DataFrame or DataFrame.apply
 
-        axis : {0 or ‘index’, 1 or ‘columns’}, default 0
-            If 0 or ‘index’: apply aggregation to each column
-            If 1 or ‘columns’: apply aggregation to each row.
+        axis : not yet supported 
         
         Returns 
         -------
-        scalar : when Series.agg is called with single aggregation
-        Series : when DataFrame.agg is called with a single aggrgation
-        DataFrame : when DataFrame.agg is called with several aggregations
+        Aggregation Result : scalar, ``Series`` or ``DataFrame``
+            When Series.agg is called with single aggregation, a scalar is returned. 
```

Comment:
Is it intended for this PR to include implementation of `Series.agg`, or just for `DataFrame`?

Reply:
it does work for both Series.agg and Dataframe.agg.